cpp/dolfinx/fem/Form.h

```diff
@@ -123,39 +122,31 @@ class Form
     for (auto& integral_type : integrals)
     {
       const IntegralType type = integral_type.first;
+      auto& kernels = integral_type.second;
```

Comment:
```suggestion
```

Reply:
`const` is implicit here since `integrals` is const.